src/components/BooksList.js

```diff
@@ -26,9 +26,9 @@ function BooksList() {
       </ul>
     );
   }
-
+  // {`${styles.bookcard} ${styles.flexContainer}`}
```

Comment:
- [ ] **[OPTIONAL]** Please don't forget to remove old pieces of commented code, they make your code base look ugly and make it less readable. 🤓 🔧